guide/src/class.md

```diff
@@ -261,8 +278,9 @@ impl SubClass {
        (SubClass{ val2: 15}, BaseClass::new())
    }
 
-   fn method2(self_: &PyClassShell<Self>) -> PyResult<usize> {
-      self_.get_super().method().map(|x| x * self_.val2)
+   fn method2(self_: PyRef<Self>) -> PyResult<usize> {
+       let super_ = self_.as_ref();  // Get &BaseClass
```

Comment:
For me this is surprising, while `get_super` was very clear.

Reply:
It was initially named `as_super` but changed to use `AsRef` since [wasm-bindgen](https://rustwasm.github.io/wasm-bindgen/web-sys/inheritance.html#accessing-parent-classes-using-asref) does so.
I prefer to `as_super` butt, in the long run, it would be beneficial to use the same API convention with a major library.